homeassistant/components/energyzero/quality_scale.yaml

```diff
@@ -0,0 +1,100 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: exempt
+    comment: |
+      It uses a coordinator.py file, but an entity.py file
+      is not used since it only has a sensor platform.
```

Comment:
I would consider this rule as `done` as all the common modules are in the right files